requirements.txt

```diff
@@ -4,89 +4,91 @@
 flask-babel==1.0.0        # via flask-appbuilder
 flask-caching==1.8.0      # via apache-superset (setup.py)
-flask-compress==1.4.0     # via apache-superset (setup.py)
+flask-compress==1.5.0     # via apache-superset (setup.py)
 flask-jwt-extended==3.24.1  # via flask-appbuilder
 flask-login==0.4.1        # via flask-appbuilder
-flask-migrate==2.5.2      # via apache-superset (setup.py)
+flask-migrate==2.5.3      # via apache-superset (setup.py)
 flask-openid==1.2.5       # via flask-appbuilder
 flask-sqlalchemy==2.4.1   # via flask-appbuilder, flask-migrate
 flask-talisman==0.7.0     # via apache-superset (setup.py)
-flask-wtf==0.14.2         # via apache-superset (setup.py), flask-appbuilder
-flask==1.1.1              # via apache-superset (setup.py), flask-appbuilder, flask-babel, flask-caching, flask-compress, flask-jwt-extended, flask-login, flask-migrate, flask-openid, flask-sqlalchemy, flask-wtf
+flask-wtf==0.14.3         # via apache-superset (setup.py), flask-appbuilder
+flask==1.1.2              # via apache-superset (setup.py), flask-appbuilder, flask-babel, flask-caching, flask-compress, flask-jwt-extended, flask-login, flask-migrate, flask-openid, flask-sqlalchemy, flask-wtf
 geographiclib==1.50       # via geopy
-geopy==1.21.0             # via apache-superset (setup.py)
+geopy==1.22.0             # via apache-superset (setup.py)
 gunicorn==20.0.4          # via apache-superset (setup.py)
-humanize==0.5.1           # via apache-superset (setup.py)
+humanize==2.4.0           # via apache-superset (setup.py)
 idna==2.9                 # via email-validator
-importlib-metadata==1.4.0  # via jsonschema, kombu
+importlib-metadata==1.6.0  # via jsonschema, kombu, markdown
 isodate==0.6.0            # via apache-superset (setup.py)
-itsdangerous==1.1.0       # via flask
-jinja2==2.10.3            # via flask, flask-babel
+itsdangerous==1.1.0       # via flask, flask-wtf
+jinja2==2.11.2            # via flask, flask-babel
 jsonschema==3.2.0         # via flask-appbuilder
-kombu==4.6.7              # via celery
-mako==1.1.1               # via alembic
-markdown==3.1.1           # via apache-superset (setup.py)
-markupsafe==1.1.1         # via jinja2, mako
+kombu==4.6.8              # via celery
+mako==1.1.2               # via alembic
+markdown==3.2.2           # via apache-superset (setup.py)
+markupsafe==1.1.1         # via jinja2, mako, wtforms
 marshmallow-enum==1.5.1   # via flask-appbuilder
-marshmallow-sqlalchemy==0.21.0  # via flask-appbuilder
-marshmallow==2.19.5       # via flask-appbuilder, marshmallow-enum, marshmallow-sqlalchemy
-more-itertools==8.1.0     # via zipp
-msgpack==0.6.2            # via apache-superset (setup.py)
-numpy==1.18.1             # via pandas, pyarrow
+marshmallow-sqlalchemy==0.23.0  # via flask-appbuilder
+marshmallow==2.21.0       # via flask-appbuilder, marshmallow-enum, marshmallow-sqlalchemy
+msgpack==1.0.0            # via apache-superset (setup.py)
```

Comment:
always a bit scared of `1.0.0` versions

Reply:
The way I see it, moving from `0.x` to `1.x` is usually a good thing in itself in the long term, and unless unit tests are picking up errors, we should be able to assume there were no relevant breaking changes.